docker/hashFolder.ts

```diff
@@ -41,6 +41,14 @@ const generateMd5ForFolder = async (
 	return md5Hashes
 }
 
+export const hashStrings = (data: string[]): string => {
```

Comment:
Duplicate of https://github.com/hello-nrfcloud/backend/blob/99ca8ddd4059584fd4b5fcf084beec78276c50c6/cdk/helpers/lambdas/checksumOfFiles.ts#L15C14-L15C31